packages/@okta/vuepress-site/docs/reference/api/user-types/index.md

```diff
@@ -212,19 +212,19 @@ curl -s -XPOST -H "Content-Type: application/json" -H "Authorization: SSWS ${api
 
 <ApiOperation method="POST" url="/api/v1/meta/types/user/{typeId}" />
 
-Updates an existing User Type. A PUT is a full replace operation; a POST is a partial update. Only the `displayName` and `description` elements can be changed; the `name` of an existing User Type cannot be changed.
+Updates an existing User Type. A PUT is a full replace operation; a POST is a partial update. Only the `displayName` and `description` elements can be changed; the `name` of an existing User Type can't be changed.
 
-The schema associated with this type is not editable with this API. If you want to edit the schema, use the [schema update API](/docs/reference/api/schemas/#add-property-to-user-profile-schema).
+The schema associated with this type isn't editable with this API. If you want to edit the schema, use the [schema update API](/docs/reference/api/schemas/#add-property-to-user-profile-schema).
 
-##### Request Parameters
+##### Request parameters
 
-The [User Type](#user-type-object) you wish to update
+The [User Type](#user-type-object) you want to update
 
 ##### Response parameters
 
-The updated [User types](#user-type-object)
+The updated [User Type](#user-type-object)
```

Comment:
shouldn't that be Types with an s?

Reply:
It's singular because the content above this sentence is the following where User Type is singular:
Updates an existing User Type. A PUT is a full replace operation. A POST is a partial update. You can only change the `displayName` and `description` elements. You can't change the `name` of an existing User Type.

Same with content in Create User Type and Get User Type.

Reply:
Oh gotcha - nevermind then. That one is fine. I see what you are saying now.